build.sh

```diff
@@ -1,49 +1,62 @@
-#!/bin/bash
+#! /usr/bin/env bash
 
 set -e # exit if a command fails
 set -o pipefail # Will return the exit status of make if it fails
 
-CFLAGS_COMMON="-O0 -g -DXBOX=1" # Compilation flags for all platforms
-POST_BUILD=""
-
 package_windows() { # Script to prepare the windows exe
-    mkdir -p dist
-    cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
-    cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
-    python2 ./get_deps.py dist/xqemu.exe dist
-    strip dist/xqemu.exe
-    strip dist/xqemuw.exe
+	mkdir -p dist
+	cp i386-softmmu/qemu-system-i386.exe dist/xqemu.exe
+	cp i386-softmmu/qemu-system-i386w.exe dist/xqemuw.exe
+	python2 ./get_deps.py dist/xqemu.exe dist
+	strip dist/xqemu.exe
+	strip dist/xqemuw.exe
```

Comment:
QEMU uses 4 spaces instead of tab. I don't think we should do this any differently in a support script, unless the language requires or strongly recommends it.

Reply:
it's ok to use space, there was just a mix of both (see the os switch and configure indentation)